frontend/src/pages/Home.jsx

```
@@ -0,0 +1,17 @@
+import React from 'react'
+import Navbar from '../components/Navbar/Navbar'
+import Footer from '../components/Footer'
+import Body from '../components/Home/Body'
+
+function Home() {
+  return (
+    <div>
+     
+      <Navbar></Navbar>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)